src/emailservice/logger.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import sys
+import json
+
+class JSONStreamHandler(logging.StreamHandler):
```

Comment:
I wonder if there's an off-the-shelf library we can use for json logging. 

It would be great to:
- keep the sample short by not writing this implementation
- not duplicate it between two python services

so I tend to think if you can find an external library, it can work better.

Reply:
Gotcha. [python-json-logger](https://github.com/madzak/python-json-logger) should support it.
I'll replace the code to use this lib.